### PR TITLE
fix: RND-56: Fix some errors with api schema

### PR DESCRIPTION
**Pull request by niklub**
Some of the parameters provided in swagger decorators are incorrect and break openapi3 validation checks.

### CODE REVIEW

File: label_studio/tests/sdk/test_tasks.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 pytestmark = pytest.mark.django_db
 from label_studio_sdk import Client
+
```
Comment (jombooth) [inline label_studio/tests/sdk/test_tasks.py]: let's revert the changes here and below in the diff view, they are causing ruff to fail

**Review by jombooth — COMMENTED** [review]
LGTM modulo getting rid of spurious imports changes

File: label_studio/tests/sdk/test_tasks.py (hunk 1 above)
Comment (niklub) [reply label_studio/tests/sdk/test_tasks.py]: idk why did these changes appear, since I haven't touch it manually but only with `make fmt-all`

Comment (jombooth) [reply label_studio/tests/sdk/test_tasks.py]: What directory are you running `make fmt-all` from? Ruff uses the following section order for dependencies:

https://docs.astral.sh/ruff/settings/#lint_isort_section-order

And I wonder if it thinks `tests` is a third-party dependency because there is no `tests` folder in the directory `ruff` is being executed from

Comment (niklub) [reply label_studio/tests/sdk/test_tasks.py]: Cool, thanks, it is actually the opposite: another `tests` folder is in the same root directory where I executed the script, so it obfuscated the third-party

**Review by jombooth — APPROVED** [review]
